upcase_booklist_app/static/main.css

```diff
@@ -233,3 +233,57 @@
     text-decoration: underline;
 }
 
+
+/* author.html */
+
+.container {
+    background-color: #ddd;
+    margin: 50px;
+    padding: 75px;
+}
+
+.author-name { 
+    display: flex;
+    flex-direction: column;
+}
+
+.name {
+    padding: 1px;
+}
+
+.author-details { 
+    display: flex;
+    flex-direction: row;
+}
+
+
+.book-detail-spacing {
+    padding-left: 2em;
+    padding:5px;
+}
+
+.book-title {
+    color: #990E16;
+    text-decoration: none;
+    font-size: 25px;
+    font-weight: bold;
+}
+
+.book-title a{
+    text-decoration: none;
+    color: #990E16;
+}
+
+.book-title a:hover{
+    color: #990E16;
+    text-decoration: underline;
+}
+
+.text {
+    font-size: 18px;
+}
+
+.book-detail-spacing {
+    padding-left: 2em;
```

Comment:
I think you'll want to use consistent units, either `em` or `px` or `rem` but not both.